codec/src/main/java/io/netty/handler/codec/compression/ZlibCodecFactory.java

```diff
@@ -33,7 +33,7 @@
     private static final boolean noJdkZlibEncoder;
 
     static {
-        noJdkZlibDecoder = SystemPropertyUtil.getBoolean("io.netty.noJdkZlibDecoder", true);
+        noJdkZlibDecoder = SystemPropertyUtil.getBoolean("io.netty.noJdkZlibDecoder", false);
```

Comment:
Yeah, I got it when reading #1481. Yes, enabling JDK by default when JDK7+ seems the right thing to do. Updating.